### ISSUE TITLE
The dev update is a USA update ,so you MUST region change.

### ISSUE THREAD

**Issue by mariogamer2** [maorninja/Guide#4]
Created this here because it's really important

Found a comment by reisyukaku :

Lol, you leaked the dev update i posted in the group without really knowing what it was. Its for US region o3ds dev systems. So im sure if people dont have the correct region secureinfo, or the right model, they might brick. Fair warning.

But it seem like it work on EUR but duplicate sys title:

Tested this with EUR O3DS, it just duplicates the system titles

See if it is needed.

**Comment by mariogamer2** [comment]
Created this here because it's really important

**Comment by mariogamer2** [comment]
Found a comment by reisyukaku : 

Lol, you leaked the dev update i posted in the group without really knowing what it was. Its for US region o3ds dev systems. So im sure if people dont have the correct region secureinfo, or the right model, they might brick. Fair warning. 

But it seem like it work on EUR but duplicate sys title:

Tested this with EUR O3DS, it just duplicates the system titles 

See if it is needed.

**Comment by NightScript370** [comment]
Okay. Thanks for posting it here.

I'll do a work-around, it also doesn't work for N3DS. By the way, reisyukaku commented here or somewhere else?

**Comment by mariogamer2** [comment]
For the EUR test,it is annother  people. For the more longer message,it's reisyukaku.

For New 3ds,I have an idea,why not writing an old 3ds ncsd in it? (OTPHelper does this,but it probably need a mod.)

**Comment by mariogamer2** [comment]
For New 3ds,I have an idea,why not writing an old 3ds ncsd in it? (OTPHelper does this,but it probably need a mod.)

**Comment by NightScript370** [comment]
Yeah. Okay. Hold off the french translation until I fix the issue

**Comment by mariogamer2** [comment]
On that ISO site

**Comment by mariogamer2** [comment]
she commented on the ISO site.
